Src/Base/AMReX_Geometry.H

```diff
@@ -68,6 +68,30 @@ public:
 };
 
     namespace detail {
+        template <typename T>
+        T bisect_prob_lo (amrex::Real plo, amrex::Real /*phi*/, amrex::Real idx, int ilo, int ihi, amrex::Real tol) {
+        T lo = static_cast<T>(plo + tol);
+        bool safe;
+        {
+            int i = int(Math::floor((lo - plo)*idx)) + ilo;
```

Comment:
This expression, which I read as "get the cell index `i` corresponding to physical coordinate `lo`" occurs multiple times throughout these functions, and presumably in the rest of the codebase, since it's the expression that the resulting value of `roundoff_lo/hi` needs to safely satisfy. Shouldn't it be in some named function, so users don't err in its details, and for clarity?

Reply:
As an aside, the convention throughout the AMReX-verse of `idx` meaning "inverse dx" rather than "index" drives me nuts

Reply:
I like the idea of a named function, but I'd prefer to change this globally in a separate PR.

Reply:
I think the use of `idx` for inverse dx in amrex is limited to just a few relatively new functions in Geometry.  In most other places, it does mean index.  So it might not be too late to change it.  In other places, we usually use `dxi` or `dxinv`.